main.rb

```diff
@@ -0,0 +1,141 @@
+require './person'
```

Comment:
- [ ] good job on creating the basic UI 👍 however your linters are set up incorrectly, the folder in your directory is named `github\workflows\linters.yml` but it is supposed to be `.github\workflows\linters.yml` the dot before `github` is missing.